logserver/src/main/java/alluxio/logserver/AlluxioLogServerProcess.java

```diff
@@ -0,0 +1,92 @@
+package alluxio.logserver;
+
+import org.apache.log4j.Hierarchy;
+import org.apache.log4j.Level;
+import org.apache.log4j.PropertyConfigurator;
+import org.apache.log4j.spi.LoggerRepository;
+import org.apache.log4j.spi.RootLogger;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.NumberFormatException;
+import java.net.*;
+import java.util.HashMap;
+import java.util.Properties;
+
+public class AlluxioLogServerProcess implements LogServerProcess {
```

Comment:
I'm not sure if we are going to use Thrift to support remote logging. If so, I'd be happy to make the change.